Marlin/src/libs/autoreport.h

```diff
@@ -25,23 +25,25 @@
 
 template <typename Helper>
 struct AutoReporter {
-  millis_t next_report_ms;
+  millis_t last_report_ms;
   uint8_t report_interval;
+  millis_t report_interval_ms;
```

Comment:
Marlin stores only the minimum data needed in SRAM because SRAM is our most precious resource. We don't need to use 4 bytes here since we can just calculate the interval when needed.